src/BuildScriptGenerator/Php/PhpBashBuildSnippet.sh.tpl

```diff
@@ -12,3 +12,9 @@ php $composer install --ignore-platform-reqs --no-interaction
 {{ else }}
 echo "No 'composer.json' file found; not running 'composer install'."
 {{ end }}
+
+{{ if NginxConfFile | IsNotBlank }}
+echo "Using the custom NGINX configuration from {{ NginxConfFile }} and followed by the `service nginx reload` command."
```

Comment:
Does NGINX even exist within our build images? I thought it was only in the [runtime images for php-fpm](https://github.com/microsoft/Oryx/blob/main/images/runtime/php-fpm/template.base.Dockerfile#L27)... I think that this change should possibly be in the [startup script generator for php](https://github.com/microsoft/Oryx/blob/main/src/startupscriptgenerator/src/php/scriptgenerator.go)?

Reply:
you are right. It should be on the run-time image. I will make the change.

Reply:
@pauld-msft made the changes on the runtime images. Also added an integration test. Your feedback is appreciated. Thanks again.